src/classes/PermanentFileSystem.ts

```diff
@@ -34,6 +34,11 @@ const isArchivePath = (requestedPath: string): boolean => (
   && requestedPath.split('/').length === 3
 );
 
+const isArchiveRootLevelFolderPath = (requestedPath: string): boolean => (
+  requestedPath.startsWith('/archives')
+  && requestedPath.split('/').length === 4
```

Comment:
```php
requestedPath.startsWith('/archives') && requestedPath.split('/').length === 4
```

For the sake of this review and perhaps for other references can you please explain the splitted path lengths, 4, 3 ... as seen here and below?

Reply:
+1, I would recommend the comments.

Reply:
I've added those comments!